.travis.yml

```diff
@@ -22,10 +22,10 @@ jobs:
       name: Terraform format
     - script: ./hack/gotest.sh
       name: Installer unit tests
+    - stage: Build
```

Comment:
Shifting the `- Stage` entry in this sequence (accidentally?) moves the Go vet and lint jobs from the "Lint & Test" stage to the "Build" stage (e.g. [here][1]).  I think we only need to update the `script: ...` line.

[1]: https://travis-ci.org/openshift/installer/builds/413861629

Reply:
yea, accidentally..